doc/voq/lag_hld.md

```diff
@@ -545,15 +546,50 @@ Shown below is the new attribute of SAI_OBJECT_TYPE_LAG object that is used for
 
 ## 8 Example Show Commands
 
+The system lag information are taken from **SYSTEM_LAG_TABLE** of **CHASSIS_APP_DB** from supervisor card.
+
+The show commands for system lags are under **show chassis** commands group as show below
+
+```
+admin@Linecard2:~$ show chassis -h
+Usage: show chassis [OPTIONS] COMMAND [ARGS]...
+
+  Chassis commands group
+
+Options:
+  -h, -?, --help  Show this message and exit.
+
+Commands:
+  modules           Show chassis-modules information
+  system-lags       Show VOQ system lags information
+  system-neighbors  Show VOQ system neighbors information
+  system-ports      Show VOQ system ports information
+admin@Linecard2:~$ 
 ```
-admin@sonic:~$ show interfaces portchannel system
-Flags: A - active, I - inactive, Up - up, Dw - Down, N/A - not available,
-       S - selected, D - deselected, * - not synced, L - local switch
-  No.  Switch ID    LAG Name      Protocol   Ports
------  ----------   -----------   ---------- --------------
-    1  0(L}         PortChannel1  SystemLAG  Slot1|Asic0|Ethernet1(S)  Slot1|Asic0|Ethernet2(S)
-    2  6            PortChannel1  SystemLAG  Slot2|Asic0|Ethernet1(S)  Slot2|Asic0|Ethernet2(S)
+### Syntax
 ```
+admin@Linecard2:~$ show chassis system-lags -h
+Usage: show chassis system-lags [OPTIONS] [SYSTEMLAGNAME]
+
+  Show VOQ system lags information
+
+Options:
+  -d, --display [all]  Show internal interfaces  [default: all]
```

Comment:
Since we are displaying internal lag information, I think we dont need the `-d` option.

Reply:
Yes. We do not need the -d option. I'll remove this.

Reply:
Fixed.